docs/enums.rst

```diff
@@ -2,36 +2,178 @@
 Dimension/Enums
 ===========
 
-Some statistics are available on a more granular level which is sumtimes
-referred to as curring the data among additional dimensions or as splitting
-it according to enums. This package sticks to the latter.
+Some statistics are available on a more granular level. This is sometimes
+referred to as curring the data among additional dimensions or as **splitting
```

Comment:
```suggestion
referred to as slicing the data (cube) across dimensions or as **splitting
```

Reply:
I'll stick with this for now. Generally the only reason I see to delve deeper into different formulations of what we are doing is when we want to draw parallels to formulations on the regionalstatistik website, the datenguide project website etc. But if it happens in the future we should put a reference otherwise it will happen again as in my initial formulation that, where it is unclear where it came from.